runner/src/server/plugins/engine/views/partials/form.html

```diff
@@ -33,6 +33,7 @@
 						<td class="govuk-table__cell">{{ row.value.text }}</td>
 						<td class="govuk-table__cell">
 							<a href={{row.action.href}}>{{ row.action.text }}</a>
+							<span class="govuk-visually-hidden">{{ row.key.text }}</span>
```

Comment:
can we move the span inside the `<a>` tag?

Reply:
Span is now within the a tag